### PR TITLE
Rework address setup in WebSocket Debugger Client

### PR DESCRIPTION
**Pull request by robertsipka**

### CODE REVIEW

File: jerry-debugger/jerry-client-ws.html [hunk 1]
```diff
@@ -972,15 +972,25 @@
...
+
+    if (ipAddr == "")
     {
-      appendLog("IP address expected");
-      return true;
+      ipAddr = "localhost";
     }
 
-    appendLog("Connect to: " + args[2]);
+    if (ipAddr.match(":")) {
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.html]: Can we use a regex here? Something like:

    /^.*:\d+$/

If match requires whole string match then ^ and $ is not needed.

Comment (robertsipka) [inline jerry-debugger/jerry-client-ws.html]: Good idea, I've changed it.

File: jerry-debugger/jerry-client-ws.html [hunk 2]
```diff
@@ -972,15 +972,25 @@
...
+
+    if (ipAddr == "")
     {
-      appendLog("IP address expected");
-      return true;
+      ipAddr = "localhost";
     }
 
-    appendLog("Connect to: " + args[2]);
+    if (ipAddr.match(/.*:\d/)) {
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.html]: Put { into the next line.

Comment (robertsipka) [inline jerry-debugger/jerry-client-ws.html]: Fixed

**Pull request merged**